kube-enforcer/templates/cluster-role.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 {{ include "aqua.labels" . | indent 4 }}
 rules:
   - apiGroups: ["*"]
-    resources: ["pods", "nodes", "namespaces", "deployments", "jobs", "cronjobs", "daemonsets", "replicasets", "replicationcontrollers", "statefulsets", "clusterroles", "clusterrolebindings", "componentstatuses", "services" ]
+    resources: ["pods", "nodes", "namespaces", "deployments", "deploymentconfigs", "jobs", "cronjobs", "daemonsets", "replicasets", "replicationcontrollers", "statefulsets", "clusterroles", "clusterrolebindings", "componentstatuses", "services" ]
```

Comment:
I think we don't need add this `deploymentconfigs` here.

Reply:
@deven0t Fixed